README.md

```diff
@@ -24,7 +24,20 @@ A simple example extracted from the unit tests:
     assertEquals(ibb, abb);
 ```
 ## Installation
-First of all, you need to install the Blosc library (visit https://github.com/Blosc/c-blosc for more details), in short, if you already have CMake, executing the following commands should do the work:
+First of all, you need to install the Blosc library (visit https://github.com/Blosc/c-blosc for more details).
+
+### Installing Blosc with Conan
+If you have [conan](https://conan.io/) installed, executing ```conan install c-blosc/v1.11.4b8@francescalted/stable``` will download you Blosc binaries.
```

Comment:
We do need such a large, specific path?  Hope that with Conan Central this would be simpler.